packages/mobile/src/identity/verification.ts

```diff
@@ -286,20 +286,28 @@ export async function requestNeededAttestations(
     numAttestationsRequestsNeeded
   )
 
-  await sendTransaction(approveTx, account, TAG, 'Approve Attestations')
-
   Logger.debug(
     `${TAG}@requestNeededAttestations`,
     `Requesting ${numAttestationsRequestsNeeded} new attestations`
   )
 
-  const requestTx = makeRequestTx(
+  const requestTx = await makeRequestTx(
     attestationsContract,
     e164NumberHash,
     numAttestationsRequestsNeeded,
     stableTokenContract
   )
-  await sendTransaction(requestTx, account, TAG, 'Request Attestations')
+
+  const {
+    confirmation: approveConfirmationPromise,
+    transactionHash: approveTransactionHashPromise,
+  } = await sendTransactionPromises(approveTx, account, TAG, 'Approve Attestations')
+
+  await approveTransactionHashPromise
```

Comment:
is this missing  `()` ?

Reply:
Should not, sendTransactionPromises returns indeed a set of promises